main.go

```diff
@@ -0,0 +1,26 @@
+package main
```

Comment:
Why do we have this file here?

Reply:
A readme file for potential users to know what this project is all about

Reply:
But this is a main.go